frontend/package.json

```diff
@@ -69,6 +71,8 @@
     "eslint-config-prettier": "6.7.0",
     "eslint-plugin-prettier": "3.1.1",
     "husky": "4.2.3",
+    "jest-environment-jsdom-fourteen": "^1.0.1",
+    "lint-staged": "10.0.8",
```

Comment:
I removed `lint-staged` in a previous PR, maybe this got in here as a merge conflict? In any case, it should be removed